### DISCUSSION TITLE
What type of system is in place now?

### DISCUSSION THREAD

**Discussion by JJ-Foust** [HACC2021/MVP#19] [Q&A] [answered]
How is the paper work tracked?
Is it a store bought solution? 
Is it a custom build?

**Comment by JJ-Foust** [comment]
How hard will it be to tie in to what is existing?

**Reply by judyn808** [reply]
The only thing I know is that FAVN results are tracked by Excel spreadsheet. I'm guessing it's a store bought solution from Windows XP times..
Regardless, Chris has indicated that they have their own software updates and she wants this queue system to be independent.

**Comment by JJ-Foust** [comment, marked as answer]
Our system will be stand alone so this doesn't matter anymore.
